test/acceptance/integration/notifications/ckan/ckan_column_http.feature

```diff
@@ -33,7 +33,7 @@ Feature: Stored in ckan new notifications per column from context broker
   Scenario Outline: stored new notifications in ckan from context broker with or without metadata
     Given copy properties.json file from "epg_properties.json" to test "ckan-sink" and sudo local "false"
     And configuration of cygnus instances with different ports "true", agents files quantity "1", id "test" and in "column" mode
-    And copy another configuration files and restart cygnus service and this execution is only once "true"
+    And copy flume-env.sh, matching table file from "default", log4j.properties, krb5.conf and restart cygnus service. This execution is only once "true"
     And verify if cygnus is installed correctly
     And "ckan" is installed correctly
     And create a new organization "<organization>" with a dataset "<service_path>"
```

Comment:
It is possible, but I think, it could need several refactorings in tests. In the future, it is something to improve.